src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+function createMessage(text, className = 'message') {
+  const div = document.createElement('div');
+
+  div.classList = className;
+  div.textContent = text;
+  document.body.append(div);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  const logo = document.querySelector('.logo');
```

Comment:
Ensure that the element with the class `.logo` exists in the DOM. If it doesn't, `logo` will be `null`, and attempting to add an event listener will result in an error.